packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -105,25 +105,30 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
 
     /// @custom:semver 1.7.2
     /// @notice Constructs the OptimismPortal contract.
+    constructor() Semver(1, 7, 2) {
```

Comment:
Semver should be updated too.

Reply:
Bumped a minor since it's still backwards-compatible, but if it warrants a major bump we can bump as well.

Reply:
Was aware of this as I left a note in the top level comment: `The bindings are regnerated. A follow up commit will bump the minor semver version in the OptimismPortal`

Reply:
Sorry, need to get better at reading the descriptions.